src/strategies_and_params/poststepstrategy.jl

```diff
@@ -193,7 +194,7 @@ is recorded.
 """
 struct Timer <: PostStepStrategy end
 
-post_step(::Timer, _) = (:time => time(),)
+post_step_action(::Timer, _, _) = (:time => time(),)
```

Comment:
Maybe it's a good idea to remove `Timer` now and always record the time in the DataFrame?

Reply:
Would this not clutter the Report?